commands/root.go

```diff
@@ -0,0 +1,105 @@
+//
+// Copyright:: Copyright 2018 Chef Software, Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	chef_load "github.com/chef/chef-load/lib"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+
+var rootCmd = &cobra.Command{
+	Use:   "chef-load",
+	Short: "A tool for simulating loading chef data",
+	Long: `A tool for simulating load on a Chef Server and/or a Chef Automate Server.
+Complete documentation is available at https://github.com/chef/chef-load`,
+	TraverseChildren: true,
+	Run:              func(cmd *cobra.Command, args []string) {},
+}
+
+// Execute adds all child commands to the root command sets flags appropriately.
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.chef-load.toml)")
+	// TODO: Should we add data_collector_url and chef_server_url to run the tool without the config?
```

Comment:
It would be nice to add the 'data_collector_url' and 'chef_server_url' to the command line also.